spec/serializers/profile_serializer_spec.rb

```diff
@@ -46,103 +37,20 @@
       }
     end
 
-    let(:court) do
-      {
-        title: court_type.title,
-        comments: 'Only speaks Spanish',
-        assessment_question_id: court_type.id,
-      }
-    end
-
-    before do
-      profile.assessment_answers = [
-        risk_alert,
-        health_alert,
-        court,
-      ]
-      profile.save!
-    end
+    let(:profile) { create(:profile, assessment_answers: [risk_alert, health_alert]) }
 
     it 'contains an `assessment_answers` nested collection' do
       expect(result[:data][:attributes][:assessment_answers].map do |alert|
         alert[:title]
-      end).to match_array [risk_alert_type.title, health_alert_type.title, court_type.title]
-    end
-  end
-
-  describe '#identifiers' do
-    let(:profile_identifiers) do
-      [
-        {
-          value: 'ABC123456',
-          identifier_type: 'police_national_computer',
-        },
-        {
-          value: 'XYZ123456',
-          identifier_type: 'prison_number',
-        },
-      ]
-    end
-
-    before do
-      profile.profile_identifiers = profile_identifiers
-      profile.save!
+      end).to match_array [risk_alert_type.title, health_alert_type.title]
     end
 
-    it 'contains two identifiers' do
-      expect(result[:data][:attributes][:identifiers]).to eql profile_identifiers
-    end
-  end
-
-  describe 'ethnicity' do
-    let(:adapter_options) { { include: { ethnicity: %I[key title description] } } }
-    let(:ethnicity) { profile&.ethnicity }
-    let(:expected_json) do
-      [
-        {
-          id: ethnicity&.id,
-          type: 'ethnicities',
-          attributes: {
-            key: ethnicity&.key,
-            title: ethnicity&.title,
-            description: ethnicity&.description,
-          },
-        },
-      ]
-    end
-
-    it 'contains an included ethnicity' do
-      expect(result[:included]).to(include_json(expected_json))
-    end
-  end
-
-  describe 'gender' do
-    before do
-      profile.update(gender_additional_information: gender_additional_information)
-    end
-
-    let(:adapter_options) { { include: { gender: %I[title description] } } }
-    let(:gender) { profile&.gender }
-    let(:gender_additional_information) { 'more info about the profile' }
-    let(:expected_json) do
-      [
-        {
-          id: gender&.id,
-          type: 'genders',
-          attributes: {
-            title: gender&.title,
-            description: gender&.description,
-          },
-        },
-      ]
-    end
-
-    it 'contains an included gender' do
-      expect(result[:included]).to(include_json(expected_json))
-    end
+    context 'with empty assessment_answers' do
```

Comment:
Given this is what a profile is, I'm not sure whether this is even a valid situation 🤔

Reply:
true, maybe we should add a validation for that, to not allow empty assessment_answers